code/_expts/slotnick_etal_2001/analyze/plots/createfigure.m

```diff
@@ -0,0 +1,30 @@
+function figure1 = createfigure(Y1, E1, taskType)
```

Comment:
`createfigure` is a bit generic; how about `de_CreateSlotnickFigure1` or somesuch?